components/docs-chef-io/content/habitat/origin-rbac.md

```diff
@@ -0,0 +1,176 @@
+    weight = 30
++++
+
+[\[edit on GitHub\]](https://github.com/habitat-sh/habitat/blob/master/components/docs-chef-io/content/habitat/builder-rbac.md)
+
+Prerequisites:
+
+- [Download the Chef Habitat CLI]({{< relref "install-habitat.md" >}})
+- [Create a Chef Habitat Builder account]({{< relref "builder-account" >}})
+- [Generate a personal access token]({{< relref "builder-profile#create-a-personal-access-token" >}})
+- [Create an origin]({{< relref "origins.md#create-origin" >}}) or accept an [invitation]({{< relref "#manage-origin-membership-with-hab-origin-invitations" >}}) to an existing origin
+- [Get origin keys]({{< relref "origins.md#origin-keys" >}})
+
+## Role-Based Access Control (RBAC) for Chef Habitat Builder (SaaS and on-prem)
+
+**New in: 1.6.140**
+{{< note >}}
+RBAC improves operational safety by letting you enable specific levels of access to each user of an origin. With this upgrade, existing origins and their users are automatically converted to RBAC. The RBAC upgrade assigns the 'Maintainer' role to existing users with a generic 'member' role. The differences between 'member' and 'Maintainer' are detailed below. The origin owner role remains unchanged.
+{{< /note >}}
+
+RBAC membership is a token-based authentication process that works at the origin level. Your membership role defines the level of access that you have to the resources within an origin. When you first join or create an origin, Chef Habitat Builder identifies your personal access token and assigns a membership role to it for that origin. By default, when you join an origin you're assigned the "read-only" role and when you create an an origin you're assigned the 'owner' role. Role access is cumulative and progressive--each RBAC role includes all of the privileges of the previous roles and adds new access privileges.
+
+RBAC Origin Member Roles:
+
+Read-Only
+: The default membership role for any user joining an origin. 'Read-Only' users can read an origin's packages, channels, origin membership, jobs, keys, integrations, invitations, roles, settings but cannot add to, change, or delete anything in the origin, including uploading packages and inviting users to the origin.
+
+Member
+: In addition to read-only access, an origin 'Member' can upload and build packages in the 'unstable' channel, but they cannot promote packages to other channels.
+
+Maintainer
+: Existing origin 'members' are now 'Maintainers'. This role has full read and write access to packages, channels, origin membership, jobs, integrations, invitations, settings. However, the 'Maintainer' role is more limited than the past role, in that 'Maintainers' only have read access to packages, channels, origin membership, jobs, keys, integrations, and settings. Origin 'Maintainers' can read origin membership roles and see and send invitations, but they cannot otherwise change origin membership--their own or anybody else's. Finally, 'Maintainers' can neither read nor write origin secrets.
```

Comment:
Might be worth calling out that "Existing origin 'members' are now 'Maintainers'." pertains to when the RBAC feature was initially rolled out. As such, it's historical information, and not current.